web/models/stable_diffusion/cache_objects.py

```diff
@@ -43,11 +47,20 @@
     model_config[args.version],
     subfolder="scheduler",
 )
-schedulers["SharkEulerDiscrete"] = SharkEulerDiscreteScheduler.from_pretrained(
-    model_config[args.version],
-    subfolder="scheduler",
-)
-schedulers["SharkEulerDiscrete"].compile()
+if args.version == "v2_1base":
+    schedulers[
+        "SharkEulerDiscrete"
+    ] = SharkEulerDiscreteScheduler.from_pretrained(
+        model_config[args.version],
+        subfolder="scheduler",
+    )
+    schedulers["SharkEulerDiscrete"].compile()
+else:
+    schedulers["SharkEulerDiscrete"] = EulerDiscreteScheduler.from_pretrained(
```

Comment:
I don't see why the SharkEulerDiscrete scheduler is exclusive to v2.1base.

Reply:
Fixed. Thanks!